var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -165,6 +167,9 @@ class Trilinos(CMakePackage):
     conflicts('+fortrilinos', when='~fortran')
     conflicts('+fortrilinos', when='@:99')
     conflicts('+fortrilinos', when='@master')
+    conflicts('+superlu-dist', when='+superlu')
+    conflicts('+superlu-dist', when='@11.14.1:11.14.3')
+    conflicts('+pnetcdf', when='@11.14.1:12.10.1')
```

Comment:
So these variants are supported prior to 11.14.1? I think you want to remove the lower bound. You never know when someone will add an older version.